cahee/css/styles.css

```diff
@@ -0,0 +1,423 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v7.0.29,
+* Autoprefixer: v9.7.6
+* Browsers: last 4 version
+*/
+
+@import url("./styles_cahee_header.css");
+@import url("./styles_cahee_blog_articles.css");
+
+.page {
+  height: 100vh;
+  width: 100vw;
+}
+
+.page--small {
+  height: 70vh;
+}
+
+.page--first {
+  background-image: url(../img/img1.png);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.page--second {
+  background-image: url(../img/img2.png);
+  background-position: 10%;
+  background-color: #00c4c4;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.page--third {
+  background-color: #b07cf7;
+}
+
+.page--fourth {
+  background-image: url(../img/contact_background.png);
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.page--service {
+  background-image: url(../img/img3.png);
+  background-position: 90%;
+  background-size: fill;
+  background-repeat: no-repeat;
+  background-color: #70bfff;
+}
+
+.page--gallery {
+  background-color: #ffe92f;
+}
+
+.page--subscribe {
+  background-image: url(../img/img4.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 50%;
+}
+
+.page__content {
+  height: 100%;
+  margin-left: 10%;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.page__content--row {
+  flex-direction: row;
```

Comment:
As soon as this element has no `display: flex`,  `flex-drirection`, `justify-content`, `align-items` do not change anything

Reply:
Since this is a modifier class for page__content class which has display: flex this class modifies flex-direction (base flex-direction for page__content is column) so this line change behaviour for a specific page.

Reply:
Got it, thanks